### ISSUE TITLE
[BUG]: What is Login Email and Password?

### ISSUE THREAD

**Issue by Quantatirsk** [activepieces/activepieces#2583]
Hi, I deployed this project using docker compose, but in your documentation no default login email and password was provided.

**Comment by abuaboud** [comment]
Hi @Quantatirsk 

You should be redirected to sign up page, Isn't that the case?

**Comment by Quantatirsk** [comment]
Thanks for your reply but when I sign up it only gives a hint like this:
![image](https://github.com/activepieces/activepieces/assets/93696669/eb801cb4-f7a8-40de-814f-2ffd783b808a)

**Comment by abuaboud** [comment]
@Quantatirsk Sorry Just noticed this, It seems user already exists? did you run in development mode before? or have user already registered?

**Comment by wescrockett** [comment]
I have this same issue. Upon installing (in Unraid with Redis and Postgres15). When registering, it says all emails (that I try) are already registered... then, when you refresh the page, it shows that signing up is disabled.

No way to do initial log in.

**Comment by abuaboud** [comment]
@wescrockett I heard this issue multiple times from the template that is created in unraid, I am not familiar with unraid 

Can you please book a call, let's debug it and try to open pr back to unraid to fix this issue

https://cal.com/abuaboud/talk

**Comment by wescrockett** [comment]
HI @abuaboud,

I ended up getting past the issue. It was a bizarre one though.

- After installation, the first load of the UI brings up the create-user page. Upon trying, it returns the message that the email was already taken.
- During some troubleshooting, guided by a mod on the Discord community, I checked the Postgres DB and did, in fact, find the user I registered in there
- I was able to log in with the credentials I supplied on the create-user page, despite the error showing that the email address was already taken.
